parse-maintainer.py

```diff
@@ -104,5 +204,24 @@ def parse_issue(md):
     output_file = f"{username}.json"
     with open(output_file, 'w', encoding='utf-8') as f:
         f.write(json_output)
+        f.write("\n")  # Add trailing newline
```

Comment:
```suggestion
        f.write("\n")
```

I suppose the comment isn't necessary :) this line is easy enough to understand.